requirements.txt

```diff
@@ -1,4 +1,5 @@
 2to3==1.0
+configparser==4.0.2;python_version<"3.0"
```

Comment:
If `configparser` is coming from the `six` library now, do we still need this new dependency?

Reply:
I guess not. deleted.